### ISSUE TITLE
2 and 4 bar repeat symbol: playback bug

### ISSUE THREAD

**Issue by Muetzenkrieger** [musescore/MuseScore#17248]
### Issue type

General playback bug

### Bug description

The two bar repeat symbol (like % but for 2 bars) doesn't give the correct playback. Instead it functions like the single bar symbol and repeats the last bar only once.

The 4 bar repeat symbol doesn't give any playback at all.

### Steps to reproduce

1. Write 2 or 4 bars of music
2. place the 2/4 bar repeat symbol in the next bar
3. start playback

### MuseScore Version

4.0.2

### Regression

I don't know

### Operating system

Windows 10

**Comment by cbjeukendrup** [comment]
Hi! This is a duplicate of https://github.com/musescore/MuseScore/issues/10922. It's being worked on!
